cli/command/image/trust.go

```diff
@@ -202,7 +203,12 @@ func trustedPull(ctx context.Context, cli command.Cli, imgRefAndAuth trust.Image
 		if err != nil {
 			return err
 		}
-		if err := imagePullPrivileged(ctx, cli, updatedImgRefAndAuth, false, platform); err != nil {
+		if err := imagePullPrivileged(ctx, cli, updatedImgRefAndAuth, PullOptions{
+			all:      false,
```

Comment:
Not for this PR, but wondering if we need a validation step for `--all` somewhere (as it's being ignored here); i.e. is there a situation where I could've provided the `--all` flag, and no error is printed (but it's being ignored)